### PR TITLE
check that author is a string

### PR DESCRIPTION
**Pull request by gurneyalex**
and don't crash if it is e.g. a list

### CODE REVIEW

**Comment by gurneyalex** [comment]
for an example of the crash, check https://travis-ci.org/OCA/vertical-hotel/jobs/82740881

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
not sure if we want a warning or an error here.

**Comment by pedrobaeza** [comment]
I think an error is better, because the correct format is a string.

**Comment by gurneyalex** [comment]
@pedrobaeza  updated

File: pylint_odoo/test_repo/broken_module3/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Test module 2
```
Comment (dreispt) [inline pylint_odoo/test_repo/broken_module3/README.rst]: 2 or 3?

File: pylint_odoo/test_repo/broken_module3/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Broken module 3 for tests',
+    'license': 'AGPL-3',
+    'author': ['Other', 'Odoo Community Association (OCA)'],  # expected string
+    'version': '1.0',
```
Comment (dreispt) [inline pylint_odoo/test_repo/broken_module3/__openerp__.py]: Use the 5 digits, to avoid additional error count in the future.

**Comment by gurneyalex** [comment]
@dreispt done

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Comment by moylop260** [comment]
FYI travis pypi deploy require a signed git tag to upload.
Then I will to create a tag 1.0.2 for this PR merged

**Comment by moylop260** [comment]
@gurneyalex 
Thanks for PR

Tag added [1.0.2](https://github.com/OCA/pylint-odoo/releases/tag/1.0.2)

**Comment by dreispt** [comment]
@moylop260 Since it's a feature rather than a fix, IMO it should rather be 1.1.0

**Comment by moylop260** [comment]
@dreispt 
You are right!
I will change it together with next pr: https://github.com/OCA/pylint-odoo/pull/5

**Pull request merged**